junit2htmlreport/parser.py

```diff
@@ -279,6 +280,13 @@ def html(self):
             """.format(msg=tag.text(self.failure_msg),
                        fail=tag.text(self.failure))
 
+        if self.outcome() == PASSED:
```

Comment:
Can this be controlled with a command line switch?

Frankly, when I get results of my tests back, I want to see 100% passing.  If that is not the case, I just want to see what failed since that is where I need to focus.  I'd want to automatically filter out the passing tests and just keep focus on what is broken, so a command line switch to control this seems reasonable.

Folks that want the list of passing tests would still get this new functionality too

Reply:
How about setting the .passed style to display:none by default. Together with #42 this would then be configurable.

Reply:
Seems reasonable

Reply:
Okay, I changed the style.